src/jdk.compiler/share/classes/com/sun/tools/javac/tree/DocPretty.java

```diff
@@ -88,7 +94,7 @@ public void print(List<? extends DocTree> list) throws IOException {
     /**
      * Print list with separators.
      */
-    protected void print(List<? extends DocTree> list, String sep) throws IOException {
+    protected void print(List<? extends DocTree> list, char sep) throws IOException {
```

Comment:
See comment above, suggesting that we reduce access to `private`.

Reply:
Done